### PR TITLE
fix: Don't use `eval` in app scripts

### PR DESCRIPTION
**Pull request by quintesse**
It messes with arguments containing spaces

### CODE REVIEW

**Comment by quintesse** [comment]
The tests are failing because of an unrelated issue. This URL isn't accessible (anymore?): https://bintray.com/cardillo/maven/download_file?file_path=joinery%2Fjoinery-dataframe%2F1.9%2Fjoinery-dataframe-1.9-jar-with-dependencies.jar

Not exactly sure what to do about that (besides adding `@Ignore` :-) )

File: src/main/java/dev/jbang/cli/App.java [hunk 1]
```diff
@@ -133,7 +133,7 @@ private static void installScripts(String name, String scriptRef, boolean benati
 	private static void installShellScript(Path file, String scriptRef, boolean benative) throws IOException {
 		List<String> lines = Arrays.asList(
 				"#!/bin/sh",
-				"eval \"exec jbang run" + (benative ? " --native " : " ") + scriptRef + " $*\"");
+				"exec jbang run" + (benative ? " --native " : " ") + scriptRef + " \"$@\"");
```
Comment (maxandersen) [inline src/main/java/dev/jbang/cli/App.java:136]: so it looks okey but exec creates a process where eval is in-process - does that change things if we have a long running java app...will ctrl-c and similar still kill the process or can we end up with detached processes?

Comment (quintesse) [reply src/main/java/dev/jbang/cli/App.java:136]: Isn't `exec` used to replace the current process? So shouldn't that handle ctrl+Cs just fine? 🤔

Comment (quintesse) [reply src/main/java/dev/jbang/cli/App.java:136]: I've done some simple testing and ctrl+c seems to work without any problems.

**Comment by maxandersen** [comment]
> Not exactly sure what to do about that (besides adding `@Ignore` :-) )

jcenter/bintray death caused it. you can fix it by just taking any other jar hosted by https.

even just pick one from https://search.maven.org :)

**Review by maxandersen — CHANGES_REQUESTED** [review]
windows test issue should be fixed and readme.adoc refer to the old form so would be good to fix that too.

**Comment by quintesse** [comment]
CI passed

**Pull request merged**